### PR TITLE
Variety of cleanup work related to docs/cli output for setting up staker and worker

### PR DESCRIPTION
**Pull request by derekpierre**
- [x] `set-worker` -> `bond-worker`
- [x] Updated output for command execution
- [x] Documentation of prometheus optional installation (Related to #2100 )
- [x] Update documentation text output for `nucypher stake divide`
- [x] Update documentation text output for `nucypher stake collect-reward`

### CODE REVIEW

File: docs/source/guides/network_node/staking_guide.rst [hunk 1]
```diff
@@ -247,15 +247,23 @@ Once you have created one or more stakes, you can view all active stake for conn
...
+    Status          Never Made a Commitment (New Stake)
+    Restaking       Yes (Unlocked)
+    Winding Down    No
+    Unclaimed Fees  0 ETH
+    Min fee rate    0 ETH
+    --------------  -----------------------------------
+    ╒═══════╤══════════╤═════════════╤═════════════╤═══════════════╕
+    │   Idx │ Value    │   Remaining │ Enactment   │ Termination   │
+    ╞═══════╪══════════╪═════════════╪═════════════╪═══════════════╡
+    │ 	0   │ 30000 NU │      	  31 │ Jun 19 2020 │ Jul 19 2020   │
```
Comment (KPrasch) [inline docs/source/guides/network_node/staking_guide.rst:263]: Odd formatting here

File: docs/source/guides/network_node/staking_guide.rst [hunk 2]
```diff
@@ -386,42 +392,54 @@ To divide an existing stake:
...
+    Worker 0x45D33d1Ff0A7E696556f36DE697E5C92C2CCcFaE ════
+    --------------  ----------------
+    Status          Committed #18436
+    Restaking       Yes (Unlocked)
+    Winding Down    No
+    Unclaimed Fees  0 ETH
+    Min fee rate    0 ETH
+    --------------  ----------------
+    ╒═══════╤═════════════════════════════╤═════════════╤═════════════╤═══════════════╕
+    │   Idx │ Value                   	  │   Remaining │ Enactment   │ Termination   │
```
Comment (KPrasch) [inline docs/source/guides/network_node/staking_guide.rst:436]: Table formatting here - Am I just seeing work wrap?

File: docs/source/guides/network_node/staking_guide.rst (hunk 1 above)
Comment (derekpierre) [reply docs/source/guides/network_node/staking_guide.rst:263]: Seems to be weird handling of rst by github - no problem when viewed here: https://github.com/nucypher/nucypher/blob/4d3cbe27990e92b26d07716f2adbeeef9ecdb0b8/docs/source/guides/network_node/staking_guide.rst

File: docs/source/guides/network_node/staking_guide.rst (hunk 2 above)
Comment (derekpierre) [reply docs/source/guides/network_node/staking_guide.rst:436]: Same comment as above.

File: docs/source/guides/network_node/staking_guide.rst [hunk 3]
```diff
@@ -186,32 +186,39 @@ the commitment period.
...
 .. code:: bash
 
-    (nucypher)$ nucypher stake create --hw-wallet
-
-    Select staking account [0]: 0
-    Enter stake value in NU [15000]: 15000
-    Enter stake duration (30 periods minimum): 30
 
-    ============================== STAGED STAKE ==============================
+    (nucypher)$ nucypher stake create --hw-wallet
```
Comment (KPrasch) [inline docs/source/guides/network_node/staking_guide.rst:190]: Looking forward to this flag being optional.  #1439

**Comment by derekpierre** [comment]
> This might be a good PR to also resolve #2116

@vzotova and I chatted this morning about the underlying problems in #2116 and #2113 . It will likely be a more involved code change.

**Pull request merged**